Gruntfile.js

```diff
@@ -33,6 +85,7 @@ grunt.registerTask( "update-authors", function() {
 	});
 });
 
-grunt.registerTask( "default", [ "jshint", "jscs" ] );
+grunt.registerTask( "default", [ "watch" ] );
```

Comment:
I can remove it making it `grunt watch` instead, but it's pretty handy when working with Sass/LiveReload (when we get that set up). `